services/settings/pkg/store/defaults/templates.go

```diff
@@ -7,43 +7,43 @@ var (
 	// name of the notification option 'Share Received'
 	TemplateShareCreated = l10n.Template("Share Received")
 	// description of the notification option 'Share Received'
-	TemplateShareCreatedDescription = l10n.Template("Notify me when I receive a share")
+	TemplateShareCreatedDescription = l10n.Template("Notify when I have received a share")
```

Comment:
Do we even need here the `Notify when...`? Maybe something like `Received a share` would be sufficient?

Reply:
This is the image with the old data for ease of understanding the layout:
![image](https://github.com/user-attachments/assets/8593ca8a-319a-4888-8abf-cf3801dd9c24)